tests/chainer_tests/dataset_tests/tabular_tests/test_as_mode.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+import chainer
+from chainer import testing
+
+from . import dummy_dataset
```

Comment:
Could you use absolute import for the utility module? Path starting from `chainer_tests.` can be used as this: https://github.com/chainer/chainer/blob/master/tests/chainer_tests/functions_tests/pooling_tests/test_average_pooling_nd.py#L15. Ditto for other test files.

Reply:
(Note: relative imports are prohibited by the OpenStack coding style)